### PR TITLE
Ingestion of spike_depths, hexcode for ontology regions, driftmap plot

### PR DESCRIPTION
**Pull request by ttngu207**
Required changes to table definition:

Add 2 attributes to the `Unit` table:
```
    spike_sites : longblob  # array of electrode associated with each spike
    spike_depths : longblob # (um) array of depths associated with each spike
```

Add 2 attributes to the `ArchivedClustering.Unit` table:
```
        spike_sites : blob@archive_store  # array of electrode associated with each spike
        spike_depths : blob@archive_store # (um) array of depths associated with each spike
```

Add 2 attributes to the `ccf.CCFAnnotation` table:
```
@schema
class CCFAnnotation(dj.Manual):
    definition = """
    -> CCF
    -> AnnotationType
    ---
    ontology_region_id: int  
    annotation  : varchar(1024)
    index (annotation)
    color_code: varchar(6) # hexcode of the color code of this region
    """
```

Once tables are updated, running 2 fixes:
+ fix_0010: ingest spike_depths, requires data locally
+ fix_0011: ingest color hexcode

### CODE REVIEW

File: pipeline/report.py [hunk 1]
```diff
@@ -388,6 +388,20 @@ def make(self, key):
...
+class ProbeLevelDriftMap(dj.Computed):
+    definition = """
+    -> ephys.ProbeInsertion
+    -> ephys.ClusteringMethod
+    ---
+    driftmap: filepath@report_store
+    """
+
+    def make(self, key):
+        pass
```
Comment (ixcat) [inline pipeline/report.py:402]: guessing this + shell/worker hooks are planned for future PR?

File: pipeline/plot/unit_characteristic_plot.py [hunk 2]
```diff
@@ -306,6 +306,86 @@ def plot_unit_bilateral_photostim_effect(probe_insertion, clustering_method=None
     return fig
 
 
+def plot_driftmap(probe_insertion, clustering_method=None, shank_no=1):
```
Comment (ixcat) [inline pipeline/plot/unit_characteristic_plot.py:309]: does this one need docs too?
(can defer this on worker components implementation as noted below if that makes more sense)

File: pipeline/report.py (hunk 1 above)
Comment (ttngu207) [reply pipeline/report.py:402]: yes, place holder for now, will implement next

File: pipeline/plot/unit_characteristic_plot.py (hunk 2 above)
Comment (ttngu207) [reply pipeline/plot/unit_characteristic_plot.py:309]: yes will need docs for this function as well as the corresponding `report` table, not yet implemented

File: pipeline/ingest/ephys.py [hunk 3]
```diff
@@ -1228,6 +1238,26 @@ def _stat(self):
...
+        pc_features = np.where(pc_features < 0, 0, pc_features)
+
+        # ---- compute center of mass of these features (spike depths) ----
+
+        # which channels for each spike?
+        spk_feature_ind = self.data['pc_feature_ind'][self.data['spike_templates'], :]
+        # ycoords of those channels?
+        spk_feature_ycoord = ycoords[spk_feature_ind]
+        # center of mass is sum(coords.*features)/sum(features)
+        self._data['spike_depths'] = np.sum(spk_feature_ycoord * pc_features**2, axis=1) / np.sum(pc_features**2, axis=1)
```
[the remark this answers is not in the document]
Comment (ttngu207) [reply pipeline/ingest/ephys.py:1254]: yes, odd to me too, but that's exactly how the `cortex-lab/spikes` repo describes it, I'm keeping the consistency there

**Pull request merged**